address.config.js

```diff
@@ -1,6 +1,19 @@
 module.exports = {
   whitelists: {
-    'publicKey.owner': [],
-    dataToken: []
-  }
+    'publicKey.owner': [
+      '0xDF1c35c3d5178e9d98043b35a6737Bd861c191c9',
+      '0xa76Fa6837A6ffc9F123F2193717A5965c68B0cbA',
+      '0x4C84a36fCDb7Bc750294A7f3B5ad5CA8F74C4A52',
+      '0x5868f08084eFa4c7b075D6AC238229fF7fDa8669'
+    ],
+    dataToken: ['0xDd64fD4Ff847A2FBEC2596E7A58fbB439654acB5']
```

Comment:
we can remove whitelist content here to have empty arrays for publicKey.owner and dataToken
MVG portal should list everything to be able to show crost-listing capabilities of portals